src/app/login/page.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+
+export default function Login () {
+  return (
+    <>
+      <main className="bg-white w-screen">
```

Comment:
Use div instead on main, 
As main already exists in layout.tsx,
It may can issue